.github/workflows/ruby.yml

```diff
@@ -9,13 +9,11 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - ruby-version: 3.2
           - ruby-version: 3.1
           - ruby-version: 3.0
```

Comment:
Why don't we remove 3.0 down to 2.4 since they have reached EOL?

Reply:
@dblock suggested we remove 2.4 for now

Reply:
I am not opposed to removing everything that's EOL.